<launch>
  <arg name="cmd_topic" default="cmd_vel_mux/input/navi"/>
  <arg name="rgb_topic" default="/camera/rgb/image_raw"/>
  <arg name="depth_topic" default="/camera/depth/image_raw"/>
  <arg name="model_path" default='/home/haoxin/dl_models/downloads/gibson-2plus-resnet50.pth' />
  
  <node name="ddppo_controller" pkg="ddppo_ros" type="ddppo_controller.py" output="log">
    <param name="model_path" value="$(arg model_path)"/>
    <remap from="goal" to="move_base_simple/goal"/>
    <remap from="cmd" to="$(arg cmd_topic)"/>
    <remap from="rgb" to="$(arg rgb_topic)"/>
    <remap from="depth" to="$(arg depth_topic)"/>
  </node>

  <node name="egocircle" pkg="egocircle" type="ego_circle"/>
  <!--<param name="/move_base/dynamic_reconfigure" value="egocircle_node" />-->
  
</launch>
    
